Apps/MAPL_GridCompSpecs_ACG.py

```diff
@@ -184,19 +187,25 @@ def dataframe(reader, columns):
             df.append(dict(zip(columns, row)))
         return df
 
+    # Python is case sensitive, so dict lookups are case sensitive.
+    # The column names are Fortran identifiers, which are case insensitive.
+    # So all lookups in the dict below should be converted to lowercase.
+    # Aliases must be lowercase.
     column_aliases = {
-        'NAME'       : 'short_name',
-        'LONG NAME'  : 'long_name',
-        'VLOC'       : 'vlocation',
-        'UNITS'      : 'units',
-        'DIMS'       : 'dims',
-        'UNGRIDDED'  : 'ungridded_dims',
-        'PREC'       : 'precision',
-        'COND'       : 'condition',
-        'DEFAULT'    : 'default',
-        'RESTART'    : 'restart',
-        'FRIENDLYTO' : 'friendlyto',
-        'ADD2EXPORT' : 'add2export'
+        'name'       : 'short_name',
+        'long name'  : 'long_name',
+        'vloc'       : 'vlocation',
+        'units'      : 'units',
+        'dims'       : 'dims',
+        'ungridded'  : 'ungridded_dims',
+        'prec'       : 'precision',
+        'cond'       : 'condition',
+        'default'    : 'default',
+        'restart'    : 'restart',
+        'friendlyto' : 'friendlyto',
+        'add2export' : 'add2export',
+        'num_subtiles' : 'num_subtiles,',
+        'averaging_interval' : 'averaging_interval'
```

Comment:
It might be worth trying to shorten some of these longer aliases.   Perhaps

- avg intvl
- n_subtiles
- add2exp

Reply:
New aliases merged after running zero-diff test.